README.md

```diff
@@ -153,29 +211,37 @@ Now that you have created a machine learning model, you can utilize the OpenScal
+You can find a sample notebook with output for [WatsonOpenScaleMachineLearning-with-outputs.ipynb](notebooks/with-output/WatsonOpenScaleMachineLearning-with-outputs.ipynb).
+
+#### Openscale Dashboard
+
+* Go to the instance of [Watson OpenScale](https://aiopenscale.cloud.ibm.com/aiopenscale/insights) for an IBM Cloud deployment, or to your deployed instance on Cloud Pak for Data on-premise version. Choose the `Insights` tab to get an overview of your monitored deployments, Accuracy alerts, and Fairness alerts.
 
 ![WOS insights](doc/source/images/WOSinsights.png)
 
-* Click on the tile for the `Spark German Risk Deployment` and you can see graphs for `Fairness`, `Accuracy`, and `Performance (Avg. Requests/Minute)`. Click on a portion of the graph to bring up a detailed view. We'll describe the data we clicked on, yours will vary.
+* Click on the tile for the `Spark German Risk Deployment` and you can see tiles for the `Fairness`, `Accuracy`, and `Performance monitors.
+
+![OpenScale monitors](doc/source/images/wos-credit-risk-dashboard.png)
+
+* Click on one of the tiles, such as *Drift* to view details. Click on a point on the graph for more information on that particular time slice.
 
-![WOS detailed graphs](doc/source/images/WOSdetailedGraphs.png)
+![Drift monitor](doc/source/images/wos-drift-monitor.png)
 
-* You can see from the image above that, at this time, our model is receiving 4.6 requests per minute, with an accuracy of 72%. We have 13% of our credit risk (and 87% of no risk) due the fact that this individual is age 18 to 25, and 28% credit risk (and 72% of no risk) due to the fact that this individual is female. This latter statistic id flagged as `bias`.
+* You will see which types of drift were detected. Click on the number to bring up a list of transactions that led to drift.
 
-* Now click `View details`.
+![Drift transactions](doc/source/images/wos-drift-details.png)
 
-![WOS transaction detail age](doc/source/images/WOStransactionDetailAge.png)
+* Click on the `Explain` icon on the left-hand menu and you'll see a list of transactions that have been run using an algorithm to provide explainability. Choose one and click `Explain`.
 
-* We can see that, for the `age` category, 87% of 18 to 25 year olds received `No Risk` compared to 82% of 26 to 75 year olds. This is not flagged as biased.
+![Choose transaction to explain](doc/source/images/wos-explain-menu.png)
 
-* Now click on the tab marked `age` and change it to `sex`. We can see that only 71% of the females group received the outcome of `No Risk` as compared to 78% of the male group. This is flagged as `BIAS`. Now click on `View Transactions`.
+* You will see a graph showing all the most influential features with the relative weights of contribution to the *Predicted outcome*.
 
-![WOS bias](doc/source/images/WOSbias.png)
+![View feature weights](doc/source/images/wos-explain-menu.png)
```

Comment:
Same as previous image - does not show feature weights - maybe it should be this one instead: doc/source/images/wos-explainability-feature-weights.png

Reply:
good catch. thanks.